### PR TITLE
CARRY: HACK: add bootstrap node MDSD logging

### PR DESCRIPTION
**Pull request by Makdaam**
Bits and pieces required by MDSD logging

### CODE REVIEW

File: data/data/bootstrap/files/etc/mdsd.d/mdsd.env.template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+MONITORING_GCS_ENVIRONMENT={{ .LoggingConfig.Environment }}
+MONITORING_GCS_ACCOUNT={{ .LoggingConfig.Namespace }}
+MONITORING_GCS_REGION={{ .LoggingConfig.Region }}
+MONITORING_GCS_CERT_CERTFILE=/etc/mdsd.d/secret/mdsdcert.pem
+MONITORING_GCS_CERT_KEYFILE=/etc/mdsd.d/secret/mdsdcert.pem
+MONITORING_GCS_NAMESPACE=AROClusterLogs
+MONITORING_CONFIG_VERSION={{ .LoggingConfig.ConfigVersion }}
+MONITORING_USE_GENEVA_CONFIG_SERVICE=true
+MONITORING_TENANT={{ .LoggingConfig.Region }}
```
Comment (mjudeikis) [inline data/data/bootstrap/files/etc/mdsd.d/mdsd.env.template:9]: Is this right? Tenant = Region?

Comment (Makdaam) [reply data/data/bootstrap/files/etc/mdsd.d/mdsd.env.template:9]: Yes, it's the region written in short form "eastus".

Comment (Makdaam) [reply data/data/bootstrap/files/etc/mdsd.d/mdsd.env.template:9]: see https://github.com/Azure/ARO-RP/blob/master/pkg/genevalogging/genevalogging.go#L586 and https://github.com/Azure/ARO-RP/blob/master/pkg/genevalogging/genevalogging.go#L562

**Comment by jim-minter** [comment]
nice!

**Pull request merged**